index.js

```diff
@@ -62,20 +67,22 @@ module.exports = function NuxtTypeScript(moduleOptions) {
   const createRule = (test) => ({ test: test, use: [] })
 
   // Resolve .ts and .tsx file extensions
-  this.nuxt.options.extensions.push("ts", "tsx")
+  this.nuxt.options.extensions.unshift("ts", "tsx")
```

Comment:
Why use `unshift` rather than `push`?

Reply:
I had errors that I did not understand.
Now I found out.
Earlier I had it written like this:
`<script src="./index" lang="ts"></script>`
And after installing the module, I needed to write everywhere:
`<script src="./index.ts" lang="ts"></script>`

But unshift makes TS a high priority

Reply:
Very odd...surprised it does that, but ok! Thanks for the explanation 👍